src/svg/Css.svg

```diff
@@ -1,4 +1,4 @@
-<svg width="100%" viewBox="0 0 305 187" fill="none" xmlns="http://www.w3.org/2000/svg">
+<svg width="305" height="187" viewBox="0 0 305 187" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Why did you add width and height on this one and the Atom one? And leave it off the others